### PR TITLE
Fix #61: Add cryengine to the database

### LINKED ISSUE #61: Add CryEngine
See `Android` in README.md to get an idea of the contents of the PR for this issue.

### PR DESCRIPTION
**Pull request by jonathanalvares9009**
## Explanation

Added cryengine to the database. Fixes #61 
## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR does not break any part of the existing code.
- [x] The PR creator have carefully read & adher to the contributing guidelines of the project.
- [x] The PR is made to the branch that's **called** "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] Have you followed the guidelines in the Contributing document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/CSwala/awesome-computer-science/pulls) for the same tech?
- [x] Is your submission is according to the desired pattern ?
- [x] Have you placed the contents in the `Readme.md` keeping the sorting order in mind ?

Does you submission include -

- [x]	Category/Type
- [x]	Official Documnetation  
- [x]	Best YouTube creators  
- [x]	Useful Websites  
- [x]	Dedicated Github page 
- [x]	Best Courses  
- [x]	Other Tips / Hacks

### CODE REVIEW

**Review by riturajjain2000 — CHANGES_REQUESTED** [review]
Resolve merge conflicts

**Comment by jonathanalvares9009** [comment]
@riturajjain2000 resolved merge conflicts

**Review by riturajjain2000 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -511,7 +511,30 @@ https://www.geeksforgeeks.org/top-10-algorithms-in-interview-questions/<br>
...
+ https://github.com/adampagedev/Cryengine3sdk.com-Community-Game <br>
+ <br>Best Courses - <br> https://www.pluralsight.com/courses/introduction-cryengine-787 <br> https://www.udemy.com/course/learn-the-basics-of-building-levels-with-cryengine/ <br>
+ https://www.udemy.com/course/3dmotive-learn-modular-level-building-for-cryengine-with-maya/ <br> https://www.udemy.com/course/3dmotive-texturing-an-industrial-door-with-ndo-for-cryengine-3/  <br>
+ <br>Other Tips / Hacks - <br> https://www.pluralsight.com/courses/tips-tricks-cryengine-939 <br> https://www.cryengine.com/news/view/tips-and-tricks-cherrypicking-from-gamesdk <br>
+ https://www.facebook.com/crytek/posts/get-tips-tricks-and-a-tutorial-about-how-to-make-particle-effects-with-cryengine/10160719114920287/ <br>
+ <br>
+ </details>
+
+<details>
+<summary> CUDA </summary>
```
Comment (riturajjain2000) [inline README.md:534]: remove these lines
CUDA is written 2 times

File: README.md [hunk 2]
```diff
@@ -511,7 +511,30 @@ https://www.geeksforgeeks.org/top-10-algorithms-in-interview-questions/<br>
...
+=======
```
Comment (riturajjain2000) [inline README.md]: this is also left

**Review by riturajjain2000 — CHANGES_REQUESTED** [review]

**Review by riturajjain2000 — APPROVED** [review]

**Pull request merged**